.github/ariane-config.yaml

```diff
@@ -4,21 +4,16 @@ allowed-teams:
 triggers:
   /test:
     workflows:
-    - conformance-aws-cni.yaml
     - conformance-clustermesh.yaml
     - conformance-ipsec-e2e.yaml
-    - conformance-eks.yaml
     - conformance-externalworkloads.yaml
     - conformance-ginkgo.yaml
     - conformance-ingress.yaml
     - conformance-multi-pool.yaml
     - conformance-runtime.yaml
     - integration-test.yaml
-    - tests-clustermesh-upgrade.yaml
     - tests-datapath-verifier.yaml
     - tests-l4lb.yaml
-    - tests-e2e-upgrade.yaml
```

Comment:
I'm a bit surprised. https://github.com/cilium/cilium/pull/34742 removed the conformance-e2e workflows, with the reasoning that we still have the `e2e-upgrade` workflow. But now we're removing that workflow as well?

Debugging upgrade-related issues after the fact is a lot more time-consuming, and they're rarely obvious (ie. folks won't intentionally run these workflows on their PRs). So I hope we're setting the right balance here. Having said that - testing upgrade/downgrade compatibility on every step of `main` **is** wasteful, we don't actually care until the first RC is cut.

Reply:
This is fair @julianwiedmann. The initial approach was to remove all CSP and upgrade tests. I was not aware of the discussion in the mentioend PR - I am ok leaving it per PR if you think it adds any value (e.g. it is a lot more time-consuming to debug upgrade-related issues after the fact).

I will update this PR.